### PR TITLE
[wip] Adds description to addon detail page.

### PR DESCRIPTION
**Pull request by chuckharmston**
Waiting on version fixes to test.

### CODE REVIEW

File: src/templates/addon/index.html [hunk 1]
```diff
@@ -2,6 +2,10 @@
   <div class="addon" data-slug="{{ slug }}">
     <h2>Add-on: {{ this.name }}</h2>
     <p>Version: {{ this.version }}</p>
+    {% if this.description %}
```
Comment (spasovski) [inline src/templates/addon/index.html]: are descriptions optional?

File: src/templates/addon/index.html [hunk 2]
```diff
@@ -2,6 +2,10 @@
   <div class="addon" data-slug="{{ slug }}">
     <h2>Add-on: {{ this.name }}</h2>
     <p>Version: {{ this.version }}</p>
```
Comment (spasovski) [inline src/templates/addon/index.html]: nuke this line

File: src/templates/addon/index.html [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+    {% if this.description %}
+      <p>Description: {{ this.description }}</p>
+    {% endif %}
+    <p>Version: {{ this.version }}</p>
```
Comment (spasovski) [inline src/templates/addon/index.html]: `{{ this.latest_public_version.version }}`

File: src/templates/addon/index.html (hunk 1 above)
Comment (diox) [inline src/templates/addon/index.html]: yes

File: src/templates/addon/index.html (hunk 3 above)
Comment (chuckharmston) [inline src/templates/addon/index.html]: This isn't a thing yet.

**Comment by spasovski** [comment]
r+

I'm just about finished replacing this with `app_tile` but since that's more involved it may need to be rethought a bit.

**Pull request merged**